boards/xtensa/nxp_adsp_imx8ulp/nxp_adsp_imx8ulp_defconfig

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_MAIN_STACK_SIZE=3072
+
+CONFIG_SOC_SERIES_NXP_IMX8ULP=y
+CONFIG_BOARD_NXP_ADSP_IMX8ULP=y
+
```

Comment:
some of these config are not needed anymore - see a patch that clears the imx8 and imx8m config - https://github.com/zephyrproject-rtos/zephyr/pull/62974/commits/cab0759282fc96515fa93776d20469552a82eccf
can you please do the same for 8ulp?